### ISSUE TITLE
Licensing: COPYING file could be clearer

### ISSUE THREAD

**Issue by OdyX** [blupi-games/blupimania#11]
In the course of writing the debian/copyright file, I struggle to clearly identify what (from this repo) is licensed under which license, and attributed to which authors; 

* The COPYING file puts all contributors on the same level (do testers hold licensing rights?)
* The LICENSE file says "Blupimania is licensed under GPLv3" (not +)
* The LICENSE.all file has a copy of the GPLv3 (not +)

… Where https://github.com/blupi-games/blupimania-dev#licenses says:
> Blupimania and all resource files are licensed to the GPLv3+ license.

The ideal (for me as Debian packager) would be for the COPYING file to have follow the [CF-1.0](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/) format; such as:

```CF-1.0
Files: *
Copyright: 2023, Mathieu Schroeter
           1994-1996, Michaël Walz
License: GPL-3+

Files: resources/de/*
Copyright: 1995-1996, Michaël Walz
License: GPL-3+

Files: resources/en/*
Copyright: 1995-1996, David Besuchet
License: GPL-3+

License: GPL-3+
 This program is free software; you can redistribute it and/or modify          
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation; version 3 dated June, 2007, or (at
 your option) any later version.
 .
 (...)
```

etc, etc, you see the point.

In other words, for blupimania to be acceptable for Debian, I need to get (from the source archive) a trustworthy assertion of license(s) and author(s) for each file (or bundles of files).

Happy to help clarify!

**Comment by Skywalker13** [comment]
About Michael and David, the copyright is holding by Epsitec SA but I like to provide people names too (out of respect for their work).

**Comment by Skywalker13** [comment]
And of course, where the + is missing with GPLv3, it's a mistake.

**Comment by OdyX** [comment]
#12 fixes it for me!
